Undefined symbols for architecture x86_64

HI,
When I use the Package throw the error to me,
Now I use version 0.0.39, but no matter what version I use it'll throw this error:

    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_SFSafariViewController", referenced from:
          objc-class-ref in TPDirect(TPDLinePay.o)
      "_OBJC_CLASS_$_WKWebView", referenced from:
          objc-class-ref in TPDirect(TPDEasyWallet.o)
      "_OBJC_CLASS_$_WKWebViewConfiguration", referenced from:
          objc-class-ref in TPDirect(TPDEasyWallet.o)
      "_OBJC_CLASS_$_WKPreferences", referenced from:
          objc-class-ref in TPDirect(TPDEasyWallet.o)
    ld: symbol(s) not found for architecture x86_64
    clang: error: linker command failed with exit code 1 (use -v to see invocation)

Plz help me to solve it, thanks.
------------------------------------------------------------------------------------------
Here's my flutter doctor -v:

[✓] Flutter (Channel stable, v1.17.5, on Mac OS X 10.15.5 19F101, locale zh-Hant-TW)
    • Flutter version 1.17.5 at /Users/oscar790512/app/flutter
    • Framework revision 8af6b2f038 (10 months ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Users/oscar790512/Library/Android/sdk
    • Platform android-29, build-tools 29.0.2
    • ANDROID_SDK_ROOT = /usr/local/share/android-sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Android Studio (version 4.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 46.0.2
    • Dart plugin version 193.7361
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] IntelliJ IDEA Community Edition (version 2019.3.5)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 47.1.2
    • Dart plugin version 193.7361

[✓] Connected device (1 available)
    • iPhone 11 • AA261E52-3060-48E0-B502-F60570144EA1 • ios • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)

• No issues found!

--- a/android/src/main/kotlin/com/tungwang/tappayflutterplugin/TappayflutterpluginPlugin.kt
+++ b/android/src/main/kotlin/com/tungwang/tappayflutterplugin/TappayflutterpluginPlugin.kt
@@ -54,15 +54,15 @@ class TappayflutterpluginPlugin: FlutterPlugin, MethodCallHandler, ActivityAware
   }
 
   override fun onDetachedFromActivityForConfigChanges() {
-    TODO("Not yet implemented")
+//    TODO("Not yet implemented")
   }
 
   override fun onReattachedToActivityForConfigChanges(binding: ActivityPluginBinding) {
-    TODO("Not yet implemented")
+//    TODO("Not yet implemented")
   }
 
   override fun onDetachedFromActivity() {
-    TODO("Not yet implemented")
+//    TODO("Not yet implemented")
   }
 
   override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?): Boolean {

--- a/ios/tappayflutterplugin.podspec
+++ b/ios/tappayflutterplugin.podspec
@@ -22,4 +22,5 @@ A new Flutter plugin.
   s.swift_version = '5.0'
   s.vendored_frameworks = 'TPDirect.framework', 'WebKit.framework', 'SafariServices.framework'
   s.resources         = "TPDirectResource/Image/*"
+  s.frameworks = 'WebKit', 'SafariServices'
 end

